### PR TITLE
[NG] Adds displayFormat attribute to LuUserTile

### PR DESCRIPTION
**Pull request by sanlucca**
Adds a displayFormat attribute to `lu-user-tile`

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://230-pr-lucca-front-LuccaSA.surge.sh

File: packages/ng/src/app/user/tile/user-tile.component.ts [hunk 1]
```diff
@@ -11,16 +11,25 @@ import { LuUserDisplayPipe } from '../display';
...
+	 * It is set to 'lf' by default
+	 */
+	@Input() displayFormat = 'lf';
+
 	/**
 	 * IUser role to display
 	 */
 	@Input() role: string;
 
+	get userName() {
```
Comment (jefBinomed) [inline packages/ng/src/app/user/tile/user-tile.component.ts]: As it is for internal use, can you please set this method to `private`?

**Comment by jefBinomed** [comment]
except a minor comment about a method visibilty, it's ok for me :)

File: packages/ng/src/app/user/tile/user-tile.component.ts (hunk 1 above)
Comment (sanlucca) [reply packages/ng/src/app/user/tile/user-tile.component.ts]: Done

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://230-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://230-pr-lucca-front-LuccaSA.surge.sh

**Pull request merged**